products/sle/main.pm

```diff
@@ -796,6 +796,12 @@ elsif (get_var("FIPS_TS")) {
         load_applicationstests;
     }
 }
+elsif (get_var('SECURITY')) {
+    prepare_target();
+    if (check_var("SECURITY", "apparmor_status")) {
+        load_security_tests_apparmor_status;
+    }
+}
```

Comment:
I think you have to `loadtest "console/consoletest_setup"` before bunch of security console tests, that handle some console setup works like ensure permission of serial device and stop packagekit etc..
Also I think if the structure of Security test set comes from FIPS  test set, can we merge this 2 sets in one block to reduce duplication?  In a individual block is fine if it will be different with FIPS in future.

Reply:
consoletest_setup added